src/ContinuousDynamicalSystem/include/BipedalLocomotion/ContinuousDynamicalSystem/CentroidalDynamics.h

```diff
@@ -151,6 +151,13 @@ class CentroidalDynamics : public DynamicalSystem<CentroidalDynamics>
      * @return true in case of success, false otherwise.
      */
     bool setControlInput(const Input& controlInput);
+
+    /**
+     * Set the gravity vector of the dynamical system.
+     * @param gravity the value of the gravity vector used in the system dynamics.
+     * @return true in case of success, false otherwise.
+     */
+    bool setGravityVector(const Eigen::Vector3d& gravity);
```

Comment:
This can return a void and it should take as input Eigen::Ref<const Eigen::Vector3d>

Reply:
Thanks @GiulioRomualdi for the suggestion.

Done in [
6b0625e](https://github.com/ami-iit/bipedal-locomotion-framework/pull/821/commits/6b0625ebeeea900b196e2a971392c9c9cdd4007e)